app/assets/stylesheets/_entries.scss

```diff
@@ -1,48 +1,61 @@
 .new_entry_wrapper {
   background: #fff;
   box-shadow: 0px 2px 2px rgba(83, 63, 63, 0.3);
-  @include pad(10% 3%);
+  padding-bottom: 20px;
 
-  @include media($medium-screen) {
-    @include pad(5%)
+  .button {
+    width: 100%;
   }
 
-  #new_entry, .edit_entry {
+  h2 {
+    padding: 25px;
+  }
 
-    .button {
-      width: 100%;
+  .input {
+    margin: 10px 0;
+    background-color:white;
+    input {
```

Comment:
Rule declaration should be preceded by an empty line<br>Selector should have depth of applicability no greater than 2, but was 3